doc/api/fs.md

```diff
@@ -1524,11 +1524,27 @@ If `options` is a string, then it specifies the encoding. Example:
 ```js
 fs.readFile('/etc/passwd', 'utf8', callback);
 ```
+*Note*: When the path is a directory, the behavior of
+`fs.readFile()` and [`fs.readFileSync()`][] is platform-specific. On macOS, Linux
+and Windows, an error will be returned. On FreeBSD, the contents of the files
```

Comment:
Should be... It doesn't work in the GitHub web view but will work in https://nodejs.org/api/fs.html#fs_fs_readfilesync_file_options